MoltenVK/MoltenVK/GPUObjects/MVKInstance.mm

```diff
@@ -402,18 +402,18 @@
 #define ADD_INST_ENTRY_POINT(func)						ADD_ENTRY_POINT(func, VK_API_VERSION_1_0, nullptr, nullptr, false)
 #define ADD_DVC_ENTRY_POINT(func)						ADD_ENTRY_POINT(func, VK_API_VERSION_1_0, nullptr, nullptr, true)
 
+// Add a core function.
 #define ADD_INST_1_1_ENTRY_POINT(func)					ADD_ENTRY_POINT(func, VK_API_VERSION_1_1, nullptr, nullptr, false)
```

Comment:
Should this be renamed?

Reply:
What are you thinking these should be renamed to? And why just the 1.1 versions?

Reply:
Well, I assume we are now using them for 1.2 entry points as well.

Reply:
Yes. The macros set the minimum API version needed to enable the function. So those should stay at 1.1 in terms of operational content.

Are you indicating that you think something will break here if the app tries to access one of those functions after setting the `VkApplicationInfo::apiVersion` to `VK_API_VERSION_1_2`?

Or are you just suggesting that we just rename the macro to something like `ADD_INST_1_1_AND_1_2_ENTRY_POINT` to make it clear that those functions are available for both API versions?

Reply:
Don't  `VK_KHR_buffer_device_address` and `VK_KHR_create_renderpass2` add new entry points?

Reply:
Yes. They're already in there as `ADD_DVC_1_2_PROMOTED_ENTRY_POINT` entries.

Reply:
Oh wait, I'm being silly. This is for new functions that were *not* part of an existing extension. Got it.